### ISSUE TITLE
Submit to Logstash-Plugins

### ISSUE THREAD

**Issue by lukewaite** [lukewaite/logstash-input-cloudwatch-logs#5]
https://www.elastic.co/guide/en/logstash/current/submitting-plugin.html

**Comment by reedflinch** [comment]
Hey Luke, any plans to submit this as an official plugin?

Thanks so much for this project, it's already extremely useful for me and my team!

**Comment by edperry** [comment]
yes it would be nice to have this published as a Gem to make the install simple.

**Comment by lukewaite** [comment]
While I'd love to get it published as an official plugin, I've not had the time to prioritize that.  

However, it is a published gem on rubygems, so while not rolled into the elastic.co documentation, it can be installed via `logstash-plugin install logstash-input-cloudwatch_logs`

https://rubygems.org/gems/logstash-input-cloudwatch_logs
